### PR TITLE
BUG: `OrdinationResults` is now importable from `skbio` and `skbio.stats.ordination` and correctly linked from the documentation

### LINKED ISSUE #1205: pcoa documentation doesn't contain a "Returns" section
I also can't find the docs for ``OrdinationResults`` - are those being linked from anywhere?

### PR DESCRIPTION
**Pull request by jairideout**
Fixes #1205.

### CODE REVIEW

File: skbio/stats/ordination/__init__.py [hunk 1]
```diff
@@ -118,9 +127,10 @@
 from ._correspondence_analysis import ca
 from ._canonical_correspondence_analysis import cca
 from ._principal_coordinate_analysis import pcoa
+from ._ordination_results import OrdinationResults
 from ._utils import (mean_and_std, scale, svd_rank, corr, e_matrix, f_matrix)
 
-__all__ = ['ca', 'rda', 'cca', 'pcoa',
+__all__ = ['ca', 'rda', 'cca', 'pcoa', 'OrdinationResults',
```
Comment (ebolyen) [inline skbio/stats/ordination/__init__.py]: :+1:

File: skbio/stats/ordination/tests/test_ordination_results.py [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+import unittest
+
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import numpy.testing as npt
+import pandas as pd
+from IPython.core.display import Image, SVG
+
+from skbio import OrdinationResults
```
Comment (ebolyen) [inline skbio/stats/ordination/tests/test_ordination_results.py]: :+1:

**Comment by ebolyen** [comment]
Thanks @jairideout!

**Comment by jairideout** [comment]
Sweet, thanks for reviewing @ebolyen!

**Pull request merged**